poms/bom/pom.xml

```diff
@@ -353,17 +353,29 @@
                         <groupId>commons-logging</groupId>
                         <artifactId>commons-logging</artifactId>
                     </exclusion>
+                    <exclusion>
+                        <groupId>io.github.classgraph</groupId>
+                        <artifactId>classgraph</artifactId>
+                    </exclusion>
                     <exclusion>
                         <groupId>javax.validation</groupId>
                         <artifactId>validation-api</artifactId>
                     </exclusion>
+                    <exclusion>
+                        <groupId>org.apache.kafka</groupId>
+                        <artifactId>kafka-clients</artifactId>
+                    </exclusion>
```

Comment:
Quarkus is managing this one already https://github.com/quarkusio/quarkus/blob/main/bom/application/pom.xml#L4598-L4602 I wonder why my new check did not fail?